examples/k3s/test-karpenter.yaml

```
@@ -0,0 +1,30 @@
+  name: test-karpenter
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test-karpenter
+  template:
+    metadata:
+      labels:
+        app: test-karpenter
+  template:
```

YAMLlint (1.38.0)

[error] duplication of key "template" in mapping

(key-duplicates)